src/main/java/org/folio/service/manager/exportmanager/ExportManagerImpl.java

```diff
@@ -78,15 +78,14 @@ protected void exportBlocking(ExportPayload exportPayload) {
     List<String> identifiers = exportPayload.getIdentifiers();
     FileDefinition fileExportDefinition = exportPayload.getFileExportDefinition();
     OkapiConnectionParams params = exportPayload.getOkapiConnectionParams();
-
     SrsLoadResult srsLoadResult = loadSrsMarcRecordsInPartitions(identifiers, params);
     exportService.export(srsLoadResult.getUnderlyingMarcRecords(), fileExportDefinition);
     List<JsonObject> instances = loadInventoryInstancesInPartitions(srsLoadResult.getInstanceIdsWithoutSrs(), params);
     List<String> mappedMarcRecords = mappingService.map(instances);
     exportService.export(mappedMarcRecords, fileExportDefinition);
 
     if (exportPayload.isLast()) {
-      exportService.postExport(fileExportDefinition);
+      exportService.postExport(fileExportDefinition, params.getTenantId());
```

Comment:
what if the postExport fails? There should be some exception handling to mark it as error?

Reply:
It is there already at a higher level. See ExportManagerImpl:expotData

Reply:
If _postExport_  throws an exception, then it triggers WorkerExecutor to call _ExportManagerImpl.handleExportResult_ with failed asyncResult